### DISCUSSION TITLE
How do Telegram bots receive messages when they are not pulled into a group of robots ？

### DISCUSSION THREAD

**Discussion by YC946586** [eternnoir/pyTelegramBotAPI#1792] [Q&A] [answered]
I found that a robot can be @ in a group that has not been pulled into the group chat, and can receive and process messages after @. I want to know how to achieve this

**Comment by Badiboy** [comment, marked as answer]
https://core.telegram.org/bots/api#inline-mode

**Reply by YC946586** [reply]
Thank you very much for your help
